openreview/venue_request/process/deployProcess.py

```diff
@@ -30,12 +30,12 @@ def process(client, note, invitation):
 - Venue home page: {baseurl}/group?id={conference_id}
 - Venue Program Chairs console: {baseurl}/group?id={program_chairs_id}
 
-If you need to make a change to the information provided in your request form, please feel free to revise it directly using the "Revision" button. You can also control several stages of your venue by using the Stage buttons. Note that any change you make will be immediately applied to your venue.  
+If you need to make a change to the information provided in your request form, please feel free to revise it directly using the "Revision" button. You can also control several stages of your venue by using the Stage buttons. Note that any change you make will be immediately applied to your venue.
```

Comment:
Same here

Reply:
I'm not sure if need to change something here.